deployment-templates/compute-engine/explain-instance.py

```diff
@@ -114,18 +129,19 @@ def GenerateConfig(context):
 # Install fluentd if necessary
 FLUENTD=$(ls /usr/sbin/google-fluentd)
 if [ -z "$FLUENTD" ]; then
-      cd $USER_HOME
-      curl -sSO https://dl.google.com/cloudagents/install-logging-agent.sh
-      bash install-logging-agent.sh
+    cd $USER_HOME
+    curl -sSO https://dl.google.com/cloudagents/install-logging-agent.sh
+    bash install-logging-agent.sh
 fi
 
 # Check whether Cloud SQL proxy is installed
 CLOUD_SQL_PROXY=$(ls $USER_HOME/cloud_sql_proxy)
```

Comment:
Looks like I bungled the install checker here. I'll fix it with my followup change. Should have been the following anyways

```
if ! [[ -x "/usr/bin/cloud_sql_proxy" ]]; then
...
```